app/views/bulk_actions/new.html.erb

```diff
@@ -84,6 +85,14 @@
           </span>
         </div>
 
+        <div role='tabpanel' class='tab-pane' id='SetTagsJob'>
```

Comment:
Would you consider adding test coverage for the changes to this file to `spec/views/bulk_actions/new.html.erb_spec.rb`?